docs/self-hosted/kubernetes-troubleshooting.md

```diff
@@ -0,0 +1,34 @@
+# Troubleshooting
+## Private Packagist Self-Hosted (Kubernetes)
+
+#### Private Packagist Self-Hosted update requires KOTS upgrade
+Please refer to the "[Updates](./kubernetes-maintenance.md#updates)" section in our maintenance guide.
```

Comment:
```suggestion
If the dashboard shows: 

'This version of Private Packagist Self-Hosted requires a version of KOTS that is different than what you currently have installed.'

Please refer to the "[Updates](./kubernetes-maintenance.md#updates)" section in our maintenance guide.
```
so that people find this when googling?
(extracted this from the screenshot I made in the min version PR)